keps/sig-testing/2420-reducing-kubernetes-build-maintenance/kep.yml

```diff
@@ -0,0 +1,46 @@
+title: KEP Template
+kep-number: 2420
+authors:
+  - "BenTheElder"
+owning-sig: sig-testing
+participating-sigs:
+  - sig-release
+status: provisional|implementable|implemented|deferred|rejected|withdrawn|replaced
```

Comment:
start with provisional and we can iterate while we figure out what to do with the non-applicable PRR bits

Reply:
done.